pkg/summarizer/summary.go

```diff
@@ -225,13 +126,52 @@ func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, co
 
 	var q config.DashboardQueue
 
-	log.Debug("Fetching config...")
-	var cfg configSnapshot
-	cfgAttrs, err := cfg.update(ctx, log, client, configPath, summaryPathPrefix, dashboards, &q, freq)
+	onConfigUpdate := func(cfg *snapshot.Config) error {
+		dashboards := cfg.AllDashboards()
```

Comment:
This shadows the `dashboards []string` filter passed into `Update()`, making us always process all tabs.